packages/x-codemod/README.md

```diff
@@ -70,14 +70,24 @@ npx @mui/x-codemod v6.0.0/preset-safe <path|folder>
 
 The list includes these transformers
 
+- [`adapter-change-import`](#adapter-change-import)
 - [`view-components-rename`](#view-components-rename)
 - [`view-components-rename-value-prop`](#view-components-rename)
 - [`localization-provider-rename-locale`](#localization-provider-rename-locale)
 - [`text-props-to-localeText`](#text-props-to-localeText)
 
+#### `adapter-change-import`
+
+Import the adapters from `@mui/x-date-pickers` instead of `@date-io`.
+
+```diff
+-import AdapterJalaali from '@date-io/jalaali';
++import { AdapterMomentJalaali } from '@mui/x-date-pickers-pro/AdapterMomentJalaali';
```

Comment:
Non pro package is mentioned in the text above as well 🙈 
```suggestion
+import { AdapterMomentJalaali } from '@mui/x-date-pickers/AdapterMomentJalaali';
```

Reply:
Also fixed on the migration guide